docs/guides/relationships.md

```diff
@@ -7,23 +7,43 @@ order: 4
 
 # Creating Relationships Between Lists
 
+This chapter assumes that that the reader has the code that was created in [Setup - Chapter 1](https://www.keystonejs.com/guides/new-project) and [Setup - Chapter 2](https://www.keystonejs.com/guides/add-lists).
+
 ## Pick assignee from Users collection (to-single relationship)
 
-Now, when we've created all necessary lists, let's link them together by setting up
+Let's link the Todo list and the User list together by setting up
 a `relationship`. Tweak the `assignee` field in `Todos.js` to match the following code:
 
+Import the `Relationship` field:
+```javascript
+const { Text, CalendarDay, Checkbox, Relationship } = require('@keystonejs/fields');
+```
+
+Old code:
 ```javascript
-{
+// ...
```

Comment:
```suggestion
```

Reply:
// ... was used in other places to indicate that there is more code above or below the piece of code, but if the preference is to not have them, fine by me